feat: Change the icon loading mode

zxuanhong opened the pull request:
appendIcon and createIcon icon loading mode is changed to html to facilitate css to change icon color

<!--

Thanks for creating this pull request!

Please make sure to link the issue you are closing as "Closes #issueNr". 
This helps us to understand the context of this PR.

-->
css change color
```
.djs-context-pad {
  > .group {
    .create-append-anything-append-element {
      fill: #1e80ff !important;
    }
  }
}
.djs-palette-entries {
  > .group {
    .create-append-anything-create-element {
      fill: #1e80ff !important;
    }
  }
}
```
<img width="1039" alt="image" src="https://user-images.githubusercontent.com/62740686/234444387-03f07357-c5c8-4782-ad43-b0c2a903205c.png">
<img width="551" alt="image" src="https://user-images.githubusercontent.com/62740686/234444700-fcc35b38-f065-47c9-a9fd-0ab10c799b8d.png">

Closes #5

Commit:
feat: Change the icon loading mode

appendIcon and createIcon icon loading mode is changed to html to
facilitate css to change icon color

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/bpmn-io/bpmn-js-create-append-anything?pullRequest=6) <br/>All committers have signed the CLA.

smbea commented:
Thanks for your contribution @zxiaozhou ! Could you please lint your code to match our code style and CI does not fail? Make sure `npm run lint` runs smoothly.

zxiaozhou commented:
@smbea I'll check later. Maybe I do have a problem with my local style

Commit:
feat: Change the icon loading mode
appendIcon and createIcon icon loading mode is changed to html to
facilitate css to change icon color

zxiaozhou commented:
@falko Have taken care of.

zxuanhong commented:
@smbea Have taken care of.

smbea commented on lib/create-append-anything/append-menu/AppendContextPadProvider.js:
```diff
@@ -24,7 +23,7 @@ AppendContextPadProvider.$inject = [
   'popupMenu',
   'translate',
-  'canvas'
+  'canvas',
```
This is unnecessary

zxuanhong replied:
Indeed, unnecessary things were added because of the vscode one-click fix

smbea commented on lib/create-append-anything/append-menu/AppendContextPadProvider.js:
```diff
@@ -1,17 +1,16 @@
-} from '../../icons/Icons';
+import { assign } from 'min-dash';
 
+import { appendIcon } from '../../icons/Icons';
 
 /**
  * A provider for append context pad button
  */
-export default function AppendContextPadProvider(contextPad, popupMenu, translate, canvas) {
-
+export default function AppendContextPadProvider(
+    contextPad,
+    popupMenu,
+    translate,
+    canvas
+) {
```
I don't see a reason to introduce these changes. It crowds the review and makes it harder to look at the new changes

zxuanhong replied:
Indeed, unnecessary things were added because of the vscode one-click fix

smbea reviewed (changes requested):
Seems like there are a lot of code style related changes that don't add any new value: new breaklines, adding unnecessary commas, etc. Let's try to keep this PR clean and meaningful with the feature we want to include.

I would also ask you to squash the commits because they are duplicated and use lowercase (`feat: change ...`).
